test/dotnet-vstest.Tests/VSTestTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using System;
+using System.IO;
+using FluentAssertions;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Cli.VSTest.Tests
+{
+    public class VSTestTests: TestBase
+    {
+        [Fact]
+        public void TestsFromAGivenContainerShouldRunWithExpectedOutput()
+        {
+            // Copy DotNetCoreTestProject project in output directory of project dotnet-vstest.Tests
+            string testAppName = "DotNetCoreTestProject";
+            TestInstance testInstance = TestAssetsManager.CreateTestInstance(testAppName).WithLockFiles();
+
+            string testProjectDirectory = testInstance.TestRoot;
+
+            // Build project DotNetCoreTestProject
+            new Build3Command()
+                .WithWorkingDirectory(testProjectDirectory)
+                .Execute()
+                .Should()
+                .Pass();
+
+            // Prepare args to send vstest
+            string configuration = Environment.GetEnvironmentVariable("CONFIGURATION") ?? "Debug";
+            string testAdapterPath = Path.Combine(testProjectDirectory, "bin", configuration, "netcoreapp1.0");
+            string outputDll = Path.Combine(testAdapterPath, $"{testAppName}.dll");
+            string argsForVstest = string.Concat("\"", outputDll, "\"", " --TestAdapterPath:", "\"", testAdapterPath, "\"", " --Platform:x64");
```

Comment:
Not exactly. we have to specify test dll with platform if its not build for x86. I think after our fix in test platform we may not need to specify TestAdapterPath.
@codito Am i missing something here?